### ISSUE TITLE
Reduce visual weight of docat layout overlayed on docs

### ISSUE THREAD

**Issue by dgkf** [randombenj/docat#122]
Thanks for the awesome work @randombenj 

On most displays, vertical space is a commodity. Having a floating header overlayed on the docs page can take up a considerable amount of space (especially on lower resolution displays) for what amounts to just a dropdown menu. Many docs pages themselves have their own headerbar, so this space gets eaten up quickly. I imagine that switching between docs versions usually only needs to happen once for most people browsing documentation, so it seems like a lot of space devoted to what should be an uncommon action.

Web design and ui/ux are not my usual focus, but some ideas that jump to mind to regain some of this space:
- Reduce the height (a rough draft experimenting with a super condensed version)
   ![image](https://user-images.githubusercontent.com/18220321/135925237-36567af6-dc9a-46e8-9e74-4e904ba02582.png)
- Consider keeping the menu bar in an absolute position so that it doesn't consume space while scrolling through docs
- Perhaps a floating _docat_ icon that can be clicked to expand a menu that includes the dropdown
- A (collapsible?) sidebar (another mock-up of a super condensed left panel)
   ![image](https://user-images.githubusercontent.com/18220321/135925797-fb00c55e-0a41-4d3a-8bca-0a8e0c86b201.png)
- Interpret a url parameter to hide the header entirely (especially useful for linking users to a specific version, where other versions would be irrelevant, but maybe this is a separate request).

**Comment by randombenj** [comment]
Thanks @dgkf for this input! I was also thinking of something like readthedocs has (a small version switcher at the bottom right.
I especially also like your left sidebar idea. As UI/UX is also not my speciality, maybe @BrunnerLivio you also have some ideas here?

**Comment by BrunnerLivio** [comment]
I wouldn’t go with the vertical left hand side-bar, since that’d be annoying on mobile (5% of the screen would always be occupied). 

I’d suggest going with the top version, or maybe even thinking about injecting the dropdown into the HTML you render. Unless you use an iframe, you could basically just search the rendered DOM for e.g. `<div class="docat-search"></div>` which would then give the user the option to decide where to render it. Off course that should be optional and per default use the top nav

**Comment by randombenj** [comment]
@timofurrer exactly, it would open a 'drop down/up' with all the versions to choose from
